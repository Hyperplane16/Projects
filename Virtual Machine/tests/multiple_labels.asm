FUNC LABEL 0
    MOV REG 0 VAL 25
    CAL VAL 2
    RET
FUNC LABEL 2
    PRINT REG 0
    RET
FUNC LABEL 2
    RET
