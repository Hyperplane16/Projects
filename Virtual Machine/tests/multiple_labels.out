Illegal count of 2 functions for label 2 found.
1